File: python/bc1848.py
for c in range(0, 3):
    sum = 0

    while True:
        val = input()

        if val == "caw caw":
            print(f"{sum}")
            break
        else:
            if val == "*--": sum += 4
            elif val == "--*": sum += 1
            elif val == "*-*": sum += 5
            elif val == "-*-": sum += 2
            elif val == "**-": sum += 6
            elif val == "-**": sum += 3
            elif val == "***": sum += 7
            else: sum += 0

File: cpp/bc1848.cpp
#include<iostream>

using namespace std;

int main()
{
	int sum;
	string cVal;
	
	for(int i = 0; i < 3; i++)
	{
		sum = 0;
		
		while(true)
		{
			getline(cin, cVal);
			
			if(cVal == "caw caw")
			{
				cout << sum << endl;
				break;
			}
			else
			{
				if(cVal == "*--") sum += 4;
				else if(cVal == "--*") sum += 1;
				else if(cVal == "*-*") sum += 5;
				else if(cVal == "-*-") sum += 2;
				else if(cVal == "**-") sum += 6;
				else if(cVal == "-**") sum += 3;
				else if(cVal == "***") sum += 7;
				else sum += 0;
				
			}
		}
	}
}
